### DISCUSSION TITLE
伯克利 CS188 人工智能导论 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#58] [Announcements]
# 伯克利 CS188 人工智能导论 | HackWay技术学习路线

课程名称：  Introduction to Artificial Intelligence

https://hackway.org/docs/AI/ai/cs188

**Comment by rinevard** [comment]
对在学CS188 fa18的同学的一些小提醒：

1. CS188 fa18关于贝叶斯网络的notes没有其它那些notes质量高，可以考虑看看CS188 fa23的关于贝叶斯网络的notes（更加详细）和COMP538: Introduction to Bayesian Networks（更加深入）作为补充。

2. CS188 fa18的project4中并没有让我们实现贝叶斯网络中的推断，可以先做CS188 fa23的project4的前几个与贝叶斯网络相关的题，再回来做fa18的project4.
